### DISCUSSION TITLE
PKSIM beginner-The poor fitting of the distribution phase

### DISCUSSION THREAD

**Discussion by Xiaoyang-svg** [Open-Systems-Pharmacology/Forum#1343] [Questions & Problems]
Dear all,

I am a new beginner of PkSim and have encountered some difficulties. As you can see from the picture below, my model doesn't fit well, especially the distribution phases, which looks like a straight line.  In fact, the observations were fits to a three-compartment model from healthy volunteers, not the possible one-compartment model shown here.

![image](https://user-images.githubusercontent.com/102672543/196220117-34f95fbc-b1dc-40d2-b260-eadf6a4f84c8.png)

Then, I did a lot of optimizations and tried a lot of  paramters. The silghtly better one was below, and the optimized parameters were plasma protein scale factor 1.4457, fraction unbound (plasma, reference value) 0.4948, ontogeny factor (alpha1-acid glycoprotein) 0.3895, hematocrit 0.9606. To my disappointment, the optimized hematocrit is definitely wrong for healthy volunteers, and I am at an impass again. So, I wonder if you have any idea (of commonly optimised parameters) or any suggestions as to what I should do next?

![image](https://user-images.githubusercontent.com/102672543/196224840-5d3df388-9d86-4598-bc54-33bd6d4956af.png)

Thank you for your attention

Looking forward to your reply.

**Comment by StephanSchaller** [comment]
Dear @Xiaoyang-svg , can you provide more context (Molecular weight/compound type, predicted/measured LogP, metabolism, ...)? This will help everyone to better understand the context.

If your first plot is a prediction, it is quite good.
But PBPK is a context-driven approach, so without having all/more info, it will be hard to help you.

Best,
Stephan

**Reply by Xiaoyang-svg** [reply]
Thanks a lot for your timely reply.

The details of the drug are as follows:
(1) Molecular weight: 1203.500g/mol;
(2) Compound type/pKa: Acid/11.57, Base/10.23;
(3) Log P: -0.89 or -7.2;
(4) Fraction unbound: 0.0441 in healthy volunteers, and 0.076~0.125 in critically ill patients;
(4) Metabolism: hasn't been determined yet;
(5) Excretion: total plasma clearance in healthy volunteers is 0.0270 L/h/kg.

The first figure predicted the PK data of healthy volunteers using the current PBPK model without any optimizations, and the solid lines are the predictions and the dots are the corresponding observations of two dose groups, respectively. The most important issue is that I feel this model does not reflect the feeling of multiple compartments like the classical PK analysis or PPK analysis. I hope to further optimize it, especially the distribution phase.

Thanks again!
Xiangyang

**Reply by StephanSchaller** [reply]
I still think this looks pretty decent for a prediction. Indeed, if the clearance is not determined, this might impact the curve shape depending on the location of the CL. How did you implement this so far?
Do you know if the drug is binding to any targets?
Can you elaborate on the two LogP values?

**Reply by Xiaoyang-svg** [reply]
Thanks for the reply. 

1. About the clearance
In fact, the total clearance of the drug was used to simulate renal clearance. This may not works, as the mean total urinary recovery of the drug in healthy volunteers of two dose groups was 3.7% and 8.3%, respectively. In addition, I also tried the combinations of renal clearance (0.001L/h/kg) and liver plasma clearance(0.026L/h/kg) or the combination of renal clearance (0.001L/h/kg) and biliary clearance (0.026L/h/kg), and both of them showed a poor fit of elimination phase. 

![image](https://user-images.githubusercontent.com/102672543/196387682-fa6c38cd-d30a-496b-b2e0-c1dcd0e683f6.png)

The results may indicate the drug was not metabolized or excreted by the liver or kidney, and I have to reset the renal plasma clerance as the value of total clearance. However, there is no denying that this would contradict to the major non-renal elimination processes reported in the literature. Since there is little data available for the metabolism, I actually don't know how to redefine the metabolism or elimination processes. 

2. About the drug binding pattern
As far as I know, the drug highly binds to proteins in circulation, though the exact proteins have not been identified. A previous study reported that PEPT2 mediated the uptake of the drug in renal tubular cells. Since I don't know how to set up the tubular reabsorption process, I haven't considered it yet.

3. About the two logP values
In fact, the twp logP values of -0.89 and -7.2 were predicted by ALOGPS and ChemAxon, respectively. I first chose -7.2 as my logP values, and made quite a few optimizations. However,  it stays the same or changes only within a small range after parameter optimizations. Furthermore, the values of -7.2 or -0.89 fits the same plasma concentration-time curve. I guess it may indicate that the drug wasn't sensitive to lipophilicity, so I didn't pay much attention to it.

Thanks again for you response.

**Comment by Chelton919** [comment]
Dear @Xiaoyang-svg, I think it is caused by your use of total clearance to simulate renal clearance, what is the urinary prototype recovery of this drug, if it is not a large percentage I think metabolic factors should be considered, I hope this helps, but I would also like to hear the views of others.

Best,
Chelton

**Reply by Xiaoyang-svg** [reply]
Dear Chelton,

Thank you very much for your reply. As you said,  the total clearance of the drug was used to simulate renal clearance. The mean total urinary recovery of the drug in healthy volunteers was 3.7% and 8.3% in the two dose groups. In fact, I also tried to divided the total clearance into renal clearances and non-renal clerances, with renal clerance setting at 0.001 or 0.002 L/h/kg (0.037 or 0.083 *0.0270) and non-renal clearance setting at liver clearance or biliary clearance (0.026  or 0.025L/h/kg). However, the predicted plasma concentration curve shows little signs of elimination (below). 

![image](https://user-images.githubusercontent.com/102672543/196301683-58192975-1eb2-466c-b878-6a69ce8915c2.png)

The results seems indicate that the drug wasn't metabolized by the liver or excrected by bile, which may contradict to the major non-renal elimination processes reporeted in the literature. Since there is little data available for the metabolism, I really don't know how to define other metabolism or elimination processes. Do you have any suggestions or any comments?


Best regards,
Xiaoyang
